From ff92a7d4d6430af59d5ed8d0f33b3bf958eb8726 Mon Sep 17 00:00:00 2001
From: Zebediah Figura <z.figura12@gmail.com>
Date: Wed, 23 Jun 2021 18:19:29 -0500
Subject: [PATCH] wined3d: Report a byte count of 1 for WINED3DFMT_UNKNOWN.

Allow things like wined3d_format_copy_data() to work on buffers.

Signed-off-by: Zebediah Figura <z.figura12@gmail.com>
---
 dlls/wined3d/utils.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dlls/wined3d/utils.c b/dlls/wined3d/utils.c
index 4019dd4d812..83d939e291f 100644
--- a/dlls/wined3d/utils.c
+++ b/dlls/wined3d/utils.c
@@ -83,7 +83,7 @@ static const struct wined3d_format_channels formats[] =
 {
     /*                                          size            offset
      *  format id                           r   g   b   a    r   g   b   a    bpp depth stencil */
-    {WINED3DFMT_UNKNOWN,                    0,  0,  0,  0,   0,  0,  0,  0,    0,   0,     0},
+    {WINED3DFMT_UNKNOWN,                    0,  0,  0,  0,   0,  0,  0,  0,    1,   0,     0},
     /* FourCC formats */
     {WINED3DFMT_UYVY,                       0,  0,  0,  0,   0,  0,  0,  0,    2,   0,     0},
     {WINED3DFMT_YUY2,                       0,  0,  0,  0,   0,  0,  0,  0,    2,   0,     0},
-- 
2.30.2

